contracts/deploy/018_upgrade_governor.js

```diff
@@ -0,0 +1,252 @@
+// 1. Deploy the latest governor contract and sets its ownership to the new multisig safe wallet.
+// 2. Submit a proposal on the old governor to call transferGovernance() on all governable contracts.
+// 3. Submit a proposal on the new governor to call claimGovernance() on all governable contracts.
+
+const {
+  isMainnet,
+  isFork,
+  isRinkeby,
+  isSmokeTest,
+} = require("../test/helpers.js");
+const {
+  log,
+  deployWithConfirmation,
+  withConfirmation,
+  executeProposal,
+  sendProposal,
+} = require("../utils/deploy");
+const { proposeArgs } = require("../utils/governor");
+const { getTxOpts } = require("../utils/tx");
+const addresses = require("../utils/addresses");
+
+const deployName = "018_upgrade_governor";
+
+const runDeployment = async (hre) => {
+  console.log(`Running ${deployName} deployment...`);
+
+  const { governorAddr } = await hre.getNamedAccounts();
+
+  // Signers
+  const sGovernor = await ethers.provider.getSigner(governorAddr);
+
+  const cOUSDProxy = await ethers.getContract("OUSDProxy");
+  const cVaultProxy = await ethers.getContract("VaultProxy");
+  const cChainlinkOracle = await ethers.getContract("ChainlinkOracle");
```

Comment:
The chainlink oracle contract is no longer used. We are using [OracleRouter](https://github.com/OriginProtocol/origin-dollar/blob/master/contracts/contracts/oracle/OracleRouter.sol) now.

Reply:
The OracleRouter is not governable so no need to include it.

As to the ChainlinkOracle, it is governable. Do we care leaving it under the old multi-sig? Any chance we would need to use it again in the future?

Reply:
After chatting on discord, we decided we won't bother migrating the no longer in-use ChainlinkOracle to use the new governor.